src/routers/validator.ts

```diff
@@ -36,12 +36,22 @@ const TYPES = [
     "custom"
 ];
 
+const ADDRESS_TYPES = [
+    "TransactionSigner",
+    "AssetOwner",
+    "Approver",
+    "Registrar"
+];
+
 const LOG_FILTER = ["block", "tx", ...TYPES];
 
 // FIXME:
 export const platformAddressSchema = Joi.string();
 // FIXME: PlatformAddress or AssetAddress
 const address = Joi.string();
+const addressFilter = Joi.string().regex(
+    new RegExp(`^(${ADDRESS_TYPES.join("|")})(,(${ADDRESS_TYPES.join("|")}))*$`)
+);
```

Comment:
Please make `addressFilter` invalidate if an address is not given.

Reply:
https://github.com/CodeChain-io/codechain-indexer/issues/237